### PR TITLE
helm: update version for helm-stack

### PR DESCRIPTION
**Pull request by prasunna09**

### CODE REVIEW

File: charts/incubator/hyperswitch-app/templates/producer/secret.yaml [hunk 1]
```diff
@@ -26,4 +26,7 @@ data:
   ROUTER__SECRETS__ADMIN_API_KEY: {{ .Values.application.server.secrets.admin_api_key | b64enc }}
   ROUTER__SECRETS__JWT_SECRET: {{ .Values.application.server.secrets.jwt_secret | b64enc }}
   ROUTER__SECRETS__MIGRATION_ENCRYPTION_TIMESTAMP: {{ "1685510751" | b64enc }}
+  ROUTER__CRM__TOKEN: {{ .Values.application.server.secrets.crm_token | b64enc }}
+  ROUTER__LOCKER__REDIS_TEMP_LOCKER_ENCRYPTION_KEY: {{ .Values.application.server.secrets.redis_temp_locker_encryption_key | b64enc }}
```
Comment (nitesh-balla) [inline charts/incubator/hyperswitch-app/templates/producer/secret.yaml:30]: Can we please add generic envs for router, consumer, and producer in `misc/secrets`?

File: charts/incubator/hyperswitch-sdk/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ assets
 | autoBuild.enable | bool | `false` | enable npm auto build |
 | autoBuild.forceBuild | bool | `true` | force rebuild assets even these files exist |
 | autoBuild.gitCloneParam.gitRepo | string | `"https://github.com/juspay/hyperswitch-web"` | hyperswitch-web repository |
-| autoBuild.gitCloneParam.gitVersion | string | `"0.16.7"` | hyperswitch-web repository tag |
+| autoBuild.gitCloneParam.gitVersion | string | `"0.272"` | hyperswitch-web repository tag |
```
Comment (nitesh-balla) [inline charts/incubator/hyperswitch-sdk/README.md:26]: please change this to `0.27.2`

**Review by nitesh-balla — CHANGES_REQUESTED** [review]

File: charts/incubator/hyperswitch-app/templates/producer/secret.yaml (hunk 1 above)
Comment (prasunna09) [reply charts/incubator/hyperswitch-app/templates/producer/secret.yaml:30]: yeah, we ll do the refactoring in separate PR

File: charts/incubator/hyperswitch-app/templates/consumer/configmap.yaml [hunk 3]
```diff
@@ -3,604 +3,500 @@ kind: ConfigMap
...
+  ROUTER__OPENSEARCH__AUTH__AUTH: "aws"
+  ROUTER__OPENSEARCH__AUTH__REGION: "eu-central-1"
+  ROUTER__OPENSEARCH__HOST:
+    {{ .Values.application.server.opensearch.host | quote }}
+  ROUTER__OPENSEARCH__INDEXES__DISPUTES: "hyperswitch-dispute-events"
+  ROUTER__OPENSEARCH__INDEXES__PAYMENT_ATTEMPTS: "hyperswitch-payment-attempt-events"
+  ROUTER__OPENSEARCH__INDEXES__PAYMENT_INTENTS: "hyperswitch-payment-intent-events"
+  ROUTER__OPENSEARCH__INDEXES__REFUNDS: "hyperswitch-refund-events"
+  ROUTER__PAYMENT_LINK__SDK_URL: "https://beta.hyperswitch.io/v1/HyperLoader.js"
+  ROUTER__PAYMENT_METHOD_AUTH__PM_AUTH_KEY: "AQICAHgOtO2q+CzdZZhY13M+XN6q8Yj9ThAShIuQIH9ziwez8wEvilmuZ7cPBV5+swCWgKcfAAAAkjCBjwYJKoZIhvcNAQcGoIGBMH8CAQAwegYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAzXEwIjkKl9d0hmtYQCARCATd/XNLplHrCR2S9F03b/QUY9/Sx1oYnAmVl84W99zzLcuApHbkQGGXeIsgPX/00Tr5iU4AjC6xSa0O2avrhSTyKYlxcRi0dSo6I8sjoT"
```
Comment (jagan-jaya) [inline charts/incubator/hyperswitch-app/templates/consumer/configmap.yaml:208]: remove this

File: charts/incubator/hyperswitch-app/templates/producer/configmap.yaml [hunk 4]
```diff
@@ -3,260 +3,495 @@ kind: ConfigMap
...
+  ROUTER__MASTER_DATABASE__POOL_SIZE: "3"
+  ROUTER__MULTIPLE_API_VERSION_SUPPORTED_CONNECTORS__SUPPORTED_CONNECTORS: "braintree"
+  ROUTER__OPENSEARCH__AUTH__AUTH: "aws"
+  ROUTER__OPENSEARCH__AUTH__REGION: "eu-central-1"
+  ROUTER__OPENSEARCH__HOST: {{ .Values.application.server.opensearch.host | quote }}
+  ROUTER__OPENSEARCH__INDEXES__DISPUTES: "hyperswitch-dispute-events"
+  ROUTER__OPENSEARCH__INDEXES__PAYMENT_ATTEMPTS: "hyperswitch-payment-attempt-events"
+  ROUTER__OPENSEARCH__INDEXES__PAYMENT_INTENTS: "hyperswitch-payment-intent-events"
+  ROUTER__OPENSEARCH__INDEXES__REFUNDS: "hyperswitch-refund-events"
+  ROUTER__PAYMENT_LINK__SDK_URL: "https://beta.hyperswitch.io/v1/HyperLoader.js"
```
Comment (jagan-jaya) [inline charts/incubator/hyperswitch-app/templates/producer/configmap.yaml:198]: change it to localhost

**Review by jagan-jaya — CHANGES_REQUESTED** [review]

File: charts/incubator/hyperswitch-app/templates/producer/secret.yaml (hunk 1 above)
Comment (prasunna09) [reply charts/incubator/hyperswitch-app/templates/producer/secret.yaml:30]: yes, let us do all the refactoring in a separate PR including configs.

**Review by jagan-jaya — APPROVED** [review]

**Pull request merged**